plugins/org.python.pydev.shared_core/src/org/python/pydev/shared_core/string/StringUtils.java

```diff
@@ -978,6 +978,22 @@ public static int count(String name, char c) {
         return count;
     }
 
+    public static int count(String name, String str) {
```

Comment:
Please test this in `StringUtilsTest`.

Some interesting things to check:

`count("aaa", "a")` 

`count("abc", "bc")` 

`count("ab", "ab")` 

`count("ab", "")` 

`count("ab", "c")`